frontend/app/services/EventbriteService.scala

```diff
@@ -161,13 +160,18 @@ object MasterclassEventService extends EventbriteService {
     Future.successful(MasterclassEvent(event, masterclassData))
   }
 
-  override def getEventsOrdering: Seq[String] = Nil
-  override def getEventsTagged(tag: String): Seq[RichEvent] = events.filter(_.tags.contains(tag.toLowerCase))
+  override def getFeaturedEvents: Seq[RichEvent] = Nil
+  override def getTaggedEvents(tag: String): Seq[RichEvent] = events.filter(_.tags.contains(tag.toLowerCase))
 }
 
-object MasterclassEventServiceHelpers {
+object EventbriteServiceHelpers {
   def availableEvents(events: Seq[RichEvent]): Seq[RichEvent] =
     events.filter(_.memberTickets.exists { t => t.quantity_sold < t.quantity_total } )
+
+  def getFeaturedEvents(orderedIds: Seq[String], events: Seq[RichEvent]): Seq[RichEvent] = {
+    val (orderedEvents, normalEvents) = events.partition { event => orderedIds.contains(event.id) }
+    orderedEvents.sortBy { event => orderedIds.indexOf(event.id) } ++ normalEvents.filter(!_.isSoldOut).take(4 - orderedEvents.length)
```

Comment:
Yeah, when they don't select 4 events to feature or when one of the featured events is non-live (i.e. probably already happened)